openmdao/docs/usr-guide/examples/parallel_multi_point.rst

```diff
@@ -135,7 +141,7 @@ need.
 
     st = time.time()
 
-    prob['X'] = np.random.random(size)
+    prob['X'] = np.ones(size) * 0.7
```

Comment:
per the setup warnings, X (from desvar) does not seem to be connected to anything and all the x parameters from the Plus components are likewise unconnected...   case issue?